test/v2/getStore.test.ts

```diff
@@ -39,11 +40,9 @@ apiSuite('Component: GET /api/v2/store', (appState: AppState) => {
       .put(`/api/v2/store/${syncKey}`)
       .expect(res => isSuccessfulResponse(res))
 
-    repoHash = res.body.hash
-
     // Create test files/dirs (first files)
     res = await agent
-      .post(`/api/v2/store/${syncKey}/${repoHash}`)
+      .post(`/api/v2/store/${syncKey}/${clientCheckpoints}`)
```

Comment:
I just realized, after taking another look at this, that we make the change because the tests will check for the right checkpoints returned depending on the checkpoints sent. With that said, it wouldn't make sense to simply call these variables "hashes". Unless you suggest that I remove these tests and later add them as a separate suite of test to check checkpoint correctness?